sass/_variables.scss

```diff
@@ -0,0 +1,164 @@
+
+
+/* ------------------------ *\
+		Variables
+\* ------------------------ */
+
+$base-font:			'Roboto' , Helvetica, sans-serif !default;
+$heading-font:		'Roboto Condensed', sans-serif !default;
+
+$font-size-base:		12px !default;
+$font-size-button:		11px !default;
+$font-size-toolbar:		14px !default;
+
+
+/* --------- Border Radius --------- */
+$soft-corners: 3px !default; // Set to zero to eliminate soft corners
+
+
+
+/* --------- COLOR :: Border --------- */
+
+
+
+$nav-height: 45px !default;
+$footer-height: 45px !default;
+
+$active-color: #4C5794;
+$large-icon-size: 32px;
+$normal-icon-size: 24px;
+
+/* Default (Light) Theme */
+
+$DEFAULT-main:				#fff !default;
+$DEFAULT-accent: 			#398dff !default;
+$DEFAULT-bg:				#f8f8f8 !default;
+$DEFAULT-menu-bg:			#fff !default;
+$DEFAULT-submenu-bg:		#f3f3f3 !default;
+$DEFAULT-hu-bg:				#ffffff !default;
+$DEFAULT-text:				#000 !default;
+$DEFAULT-text-subtle:		#999 !default;
+$DEFAULT-text-medium:		#666 !default;
+$DEFAULT-text-strong:		#000 !default;
+$DEFAULT-border:          #e9e9ed !default;
+$DEFAULT-border-subtle:		#ddd !default;
+$DEFAULT-border-medium:		#ccc !default;
+$DEFAULT-border-strong:		#999 !default;
+
+
+
+
+
+
+
+
+
+
+/* ------------------------ *\
+        Mixin		
+\* ------------------------ */
+
+/* --------- Animation : dialog --------- */
+@mixin dialog-transition {
```

Comment:
you can reuse `@mixin transition` here.

A note: including the same mixin that has no change in arguments a few places can result in a pretty big transpiled `css` file with many duplicate code. To save space, I think it better to use 

```scss
a, b, c {
    // ...a lot of style code.
}
```